app/signals/apps/api/views/ml_tool_proxy_v2.py

```diff
@@ -0,0 +1,28 @@
+import pickle
+
+from drf_spectacular.utils import extend_schema
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+
+from signals.apps.classification.models import Classifier
+
+
+@extend_schema(exclude=True)
+class LegacyMlPredictCategoryViewV2(APIView):
+    def post(self, request, *args, **kwargs):
+        try:
+            classifier = Classifier.objects.get(is_active=True)
+
+            main_model = pickle.load(classifier.main_model)
+            sub_model = pickle.load(classifier.sub_model)
```

Comment:
Hier zouden we denk ik wel een LRU (in-memory) cache mogen gebruiken. Dat zal de performance van het endpoint enorm verhogen. Daar kunnen we samen even naar kijken.